testsupport/junit5/src/test/java/com/adobe/testing/s3mock/junit5/S3MockExtensionProgrammaticTest.java

```diff
@@ -33,7 +33,8 @@
  */
 public class S3MockExtensionProgrammaticTest {
   @RegisterExtension
-  static final S3MockExtension S3_MOCK = S3MockExtension.builder().silent().build();
+  static final S3MockExtension S3_MOCK = S3MockExtension.builder().silent()
+          .withSecureConnection(false).build();
```

Comment:
I still wonder under which circumstances this test fails if it's using `https://` in the client config.